index.js

```diff
@@ -46,13 +53,18 @@ module.exports.sync = function (patterns, opts) {
 	var force = opts.force;
 	delete opts.force;
 
+	var test = opts.test;
+	delete opts.test;
+
 	return globby.sync(patterns, opts).map(function (file) {
 		if (!force) {
 			safeCheck(file);
 		}
 
 		file = path.resolve(opts.cwd || '', file);
-		rimraf.sync(file);
+		if(!test) {
```

Comment:
Please insert a line break before the `if (!test) {`, to be consistent with the `if (test) {` at line 39.